website/docs/r/codebuild_project.html.markdown

```diff
@@ -151,8 +151,8 @@ The following arguments are supported:
 
 `cache` supports the following:
 
-* `type` - (Required) The type of storage that will be used for the AWS CodeBuild project cache. The only valid value is `S3`.
-* `location` - (Required) The location where the AWS CodeBuild project stores cached resources. This value must be a valid S3 bucket name/prefix.
+* `type` - (Optional) The type of storage that will be used for the AWS CodeBuild project cache. Valid values: `NO_CACHE` and `S3`. Defaults to `NO_CACHE`.
+* `location` - (Required when cache type is not `NO_CACHE`) The location where the AWS CodeBuild project stores cached resources. For type `S3` the value must be a valid S3 bucket name/prefix.
```

Comment:
Might be slightly clearer to state 'Required when cache type is `S3`'. On my first read through my eyes jumped over the not.

Reply:
Great catch, updated to: Required when cache type is `S3` 👍